sql/connect/common/src/main/scala/org/apache/spark/sql/connect/client/SparkConnectClient.scala

```diff
@@ -707,12 +708,11 @@ object SparkConnectClient {
    */
   private def genUserAgent(value: String): String = {
     val scalaVersion = Properties.versionNumberString
-    val jvmVersion = System.getProperty("java.version").split("_")(0)
+    val jvmVersion = SparkSystemUtils.javaVersion.split("_")(0)
```

Comment:
If `Runtime.version` is used to obtain the Java version, then here we can use `SparkSystemUtils.javaVersion.feature`

Reply:
Oh, no. @LuciferYang . `SparkSystemUtils.javaVersion` should be `String` time. So, let's leave this for now.
```
-  val javaVersion = System.getProperty("java.version")
+  val javaVersion = Runtime.version.toString
```